### DISCUSSION TITLE
How do you manage your library?

### DISCUSSION THREAD

**Discussion by st-den** [koreader/koreader#8186] [Q&A]
Hey there!

I wonder what library management methods are available in KOReader? Is it all based on folders, or is there collections functionality somewhere hidden?

And if I may ask here, I would also like to know how do you, KOReader users, manage your libraries. MobileRead Forums are a little hard to navigate, and the info I searched for is usually outdated, so I would be glad to learn your workflows! Thank you.

**Comment by jimman2003** [comment]
Calibre, for me, though its send to device feature

**Reply by NiLuJe** [reply]
^ This ;).

**Comment by LucyOne** [comment]
On the computer I use Calibre.
I mainly read on my two Pocketbook devices. They have a nice library built in, which shows the books last opened, recently added, folders/authors sorted by name, series, file types, genres (by tags) and favourites. I mostly use the first two options to choose the books I want to read or I do a manual search for a specific book, then open it with KOReader.
I find the file browser in KOReader a bit slow for my liking, that's why I hardly use it. But I often take a peak at my history to switch between books I'm reading at the same time.

**Comment by st-den** [comment]
Finally, I spent some time organizing my library, so guess I ought to describe my setup, too. Here it goes:

- I read predominately on my Kobo e-reader, and sometimes I read on PC.
- When I get a new e-book, I either add it by hand to Calibre, or it gets added automatically from several chosen folders. Then I update the metadata.
- I have three Virtual Libraries based on a Medium column: E-reader, PC or Physical -- so I set it first.
- Then I set a Shelf column: Non-fiction, Fiction or Technical. These shelves are also sub-folders on the reader. 
- After that, I set the page count using a plugin.
- If I need to upload new books to my e-book, I connect KOReader to Calibre wirelessly. A trick I learned is not to use any Kobo plugins, because they use Nickel database to scan for books and metadata, and you don't want that if you use KOReader. Let Calibre scan files directly instead.
- Finally, when I finish a book, I archive my annotations using KoHighlights, and in Calibre I change the reading status and set a date I finished reading (or dropped it, whatever).
- If I need help choosing a book to read, I use Cover grid view. Actually, I use it even if I know what I want to read, because it simply looks too good :)

It's not perfect: for example, I don't tag books, and annotations aren't integrated in my database yet. But I'm fine with such a system for now.

**Comment by monksy** [comment]
I manage mine by syncing my ebooks from a network folder to an SD card on the device. I'm not interested in using a server to host and save the books. I want to have access to my entire collection.
